entropylab/graph_experiment.py

```diff
@@ -27,11 +28,51 @@
     ExperimentHandle,
     _Experiment,
 )
-from entropylab.api.graph import GraphHelper, Node, Output, _NodeExecutionInfo
+from entropylab.api.graph import (
+    GraphHelper,
+    Node,
+    Output,
+    _NodeExecutionInfo,
+    RetryBehavior,
+)
 from entropylab.instruments.lab_topology import ExperimentResources
 from entropylab.logger import logger
 
 
+def _handle_wait_time(wait_time, backoff, added_delay, maximum_wait_time):
+    wait_time *= backoff
```

Comment:
It seems like the backoff parameter meant to set the time the system waits until the next attempt to run. 
I think the parameter name is not fully indicative of what it does. I wanted to suggest `retry_time` but not even sure that's correct.

Reply:
I believe that backoff is a known term https://en.wikipedia.org/wiki/Exponential_backoff

Reply:
I didn't know it was, interesting. 
Still, I feel it doesn't tell the user what it does. 
I suppose they can learn,we just need a good docstring